examples/jsm/nodes/core/Node.d.ts

```diff
@@ -15,6 +15,7 @@ export class Node {
 	analyzeAndFlow( builder: NodeBuilder, output: string, settings?: object ): object;
 	flow( builder: NodeBuilder, output: string, settings?: object ): object;
 	build( builder: NodeBuilder, output: string, uuid?: string ): string;
+	generate( builder: NodeBuilder, output: string ): string;
```

Comment:
Are you sure this signature is correct? `generate()`  is used in `Node.build()` like so:
```js
return this.generate( builder, output, uuid );
```
In any event, it can't hurt to define `generate()` in `Node.js` with the intended signature and a comment, that this method needs to be implement by concrete node classes.

Reply:
Sure, thanks for the review! I will update my PR :)

Reply:
Done :)